### ISSUE TITLE
NTR: Uric acid fractional excretion decreased (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8831]
New term request Uric acid fractional excretion decreased (elthat_100528042921):

**Comment by pnrobinson** [comment]
@MickeySegal

This is probably synonymous with [Impaired renal uric acid clearance HP:0004732](https://hpo.jax.org/app/browse/term/HP:0004732) -- however, possibly we should revise the primary label of this term?

**Comment by MickeySegal** [comment]
We got "Decreased fractional excretion of urinary uric acid" from https://www.ncbi.nlm.nih.gov/books/NBK53700/ but it sounds like you are correct that your term describes the same thing.  I don't have a strong preference on which language to use in the name and which language to use in the synonym but I'm glad to follow your lead.

**Comment by pnrobinson** [comment]
@MickeySegal 
I added "Uric acid fractional excretion decreased" to be a synonym of [Impaired renal uric acid clearance HP:0004732](https://hpo.jax.org/app/browse/term/HP:0004732)
